s11/oper_array.rb

```diff
@@ -0,0 +1,60 @@
+# Multiply with asterisk symbol
+p 4*6
+p "A"*3
+p ["A","B","C"] * 2
+
+# Problem
+# Custom_multiply
+def custom_multiply(arr,n)
+  fin = []
+  n.times do
+    arr.each {|i| fin << i}
+  end
+  fin
+end
+p custom_multiply([1,2,3],3)
+puts
+
+# Union (|)
+p %w[a b c] | %w[c d e] # join and eliminates the repeated
+
+# Problem
+# Custom_union
+
+def custom_union(arr1, arr2)
+  arr1.dup.concat(arr2).uniq
```

Comment:
Why do you need to use `arr1.dup` here?

Reply:
To create a duplicate of the Array, and when attaching the `arr2`, it doesn't get contaminated.